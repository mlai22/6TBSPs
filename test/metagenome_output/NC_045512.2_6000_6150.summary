NC_045512.2_6000_6150	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	267	1.1254150391394949e-32
NC_045512.2_6000_6150	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	267	1.1254150391394949e-32
NC_045512.2_6000_6150	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	267	1.1254150391394949e-32
NC_045512.2_6000_6150	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	267	1.1254150391394949e-32
NC_045512.2_6000_6150	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	25	17.620179689225374
NC_045512.2_6000_6150	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	25	17.620179689225374
NC_045512.2_6000_6150	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	19	72.42975493417734
NC_045512.2_6000_6150	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	84.25820713051866
NC_045512.2_6000_6150	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	84.25820713051866
NC_045512.2_6000_6150	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	84.25820713051866
NC_045512.2_6000_6150	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	84.25820713051866
NC_045512.2_6000_6150	YP_009724397.2:1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	84.25820713051866
NC_045512.2_6000_6150	YP_009724393.1:1-222 membrane glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	315.96827673944495
NC_045512.2_6000_6150	YP_009724397.2:1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	435.4477527980108
NC_045512.2_6000_6150	YP_009725300.1 nsp4 [polyprotein_range=YP_009724389.1:2764-3263] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	435.4477527980108
NC_045512.2_6000_6150	YP_009725301.1 3C-like proteinase [polyprotein_range=YP_009724389.1:3264-3569] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	435.4477527980108
NC_045512.2_6000_6150	YP_009742611.1 nsp4 [polyprotein_range=YP_009725295.1:2764-3263] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	435.4477527980108
NC_045512.2_6000_6150	YP_009742612.1 3C-like proteinase [polyprotein_range=YP_009725295.1:3264-3569] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	435.4477527980108
NC_045512.2_6000_6150	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	496.217924783984
NC_045512.2_6000_6150	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	496.217924783984
NC_045512.2_6000_6150	YP_009725308.1 helicase [polyprotein_range=YP_009724389.1:5325-5925] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	496.217924783984
NC_045512.2_6000_6150	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	496.217924783984
NC_045512.2_6000_6150	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	1543.7890993279502
NC_045512.2_6000_6150	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	1543.7890993279502
NC_045512.2_6000_6150	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	1960.349231111099
