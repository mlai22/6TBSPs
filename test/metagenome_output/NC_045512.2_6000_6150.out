Frame: 2
Query: DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP
Length: 49
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 267
E-value: 1.1254150391394949e-32

0     DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  48  
      |||||||||||||||||||||||||||||||||||||||||||||||||      
1912  DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  1960


Frame: 2
Query: DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP
Length: 49
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 267
E-value: 1.1254150391394949e-32

0     DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  48  
      |||||||||||||||||||||||||||||||||||||||||||||||||      
1912  DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  1960


Frame: 2
Query: DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP
Length: 49
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 267
E-value: 1.1254150391394949e-32

0     DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  48  
      |||||||||||||||||||||||||||||||||||||||||||||||||      
1094  DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  1142


Frame: 2
Query: DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP
Length: 49
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 267
E-value: 1.1254150391394949e-32

0     DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  48  
      |||||||||||||||||||||||||||||||||||||||||||||||||      
1094  DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP  1142


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 25
E-value: 17.620179689225374

3     NVTLSSLEAGF  13  
      **|***|**||      
3255  SITSAVLQSGF  3265


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 25
E-value: 17.620179689225374

3     NVTLSSLEAGF  13  
      **|***|**||      
3255  SITSAVLQSGF  3265


Frame: -3
Query: REKCNFKLS
Length: 9
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 19
E-value: 72.42975493417734

2     KCNFKL  7   
      |*|*||      
3838  KLNIKL  3843

5     FKLS  8   
      ||||      
5468  FKLS  5471


Frame: -2
Query: QGKM
Length: 4
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 16
E-value: 84.25820713051866

0    QGK  2  
     |||     
955  QGK  957


Frame: -2
Query: QGKM
Length: 4
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 16
E-value: 84.25820713051866

0    QGK  2  
     |||     
955  QGK  957


Frame: -2
Query: QGKM
Length: 4
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 84.25820713051866

0    QGK  2  
     |||     
137  QGK  139


Frame: -2
Query: QGKM
Length: 4
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 84.25820713051866

0    QGK  2  
     |||     
137  QGK  139


Frame: -2
Query: QGKM
Length: 4
Subject: YP_009724397.2:1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 419

Score: 16
E-value: 84.25820713051866

1   GKM  3  
    |||     
98  GKM  100


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009724393.1:1-222 membrane glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 222

Score: 16
E-value: 315.96827673944495

12  GFL  14
    |||    
24  GFL  26


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009724397.2:1-419 nucleocapsid phosphoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 419

Score: 15
E-value: 435.4477527980108

0    SGK  2  
     |||     
234  SGK  236


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009725300.1 nsp4 [polyprotein_range=YP_009724389.1:2764-3263] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 500

Score: 15
E-value: 435.4477527980108

0    SGK  2  
     |||     
162  SGK  164


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009725301.1 3C-like proteinase [polyprotein_range=YP_009724389.1:3264-3569] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 306

Score: 15
E-value: 435.4477527980108

0  SGK  2 
   |||    
9  SGK  11


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009742611.1 nsp4 [polyprotein_range=YP_009725295.1:2764-3263] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 500

Score: 15
E-value: 435.4477527980108

0    SGK  2  
     |||     
162  SGK  164


Frame: -1
Query: SGKNVTLSSLEAGFL
Length: 15
Subject: YP_009742612.1 3C-like proteinase [polyprotein_range=YP_009725295.1:3264-3569] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 306

Score: 15
E-value: 435.4477527980108

0  SGK  2 
   |||    
9  SGK  11


Frame: -3
Query: REKCNFKLS
Length: 9
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 13
E-value: 496.217924783984

6     KLS  8   
      |||      
2655  KLS  2657


Frame: -3
Query: REKCNFKLS
Length: 9
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 13
E-value: 496.217924783984

6     KLS  8   
      |||      
1837  KLS  1839


Frame: -3
Query: REKCNFKLS
Length: 9
Subject: YP_009725308.1 helicase [polyprotein_range=YP_009724389.1:5325-5925] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 601

Score: 13
E-value: 496.217924783984

6    KLS  8  
     |||     
145  KLS  147


Frame: -3
Query: REKCNFKLS
Length: 9
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 13
E-value: 496.217924783984

6     KLS  8   
      |||      
1837  KLS  1839


Frame: 3
Query: ILYQTNHIQTQASIILSLYVIISNLLMI
Length: 28
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 13
E-value: 1543.7890993279502

25    LMI  27  
      |||      
5245  LMI  5247


Frame: 3
Query: ILYQTNHIQTQASIILSLYVIISNLLMI
Length: 28
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 13
E-value: 1543.7890993279502

25   LMI  27 
     |||     
853  LMI  855


Frame: 2
Query: DLVPNQPYPNASFDNFKFVCDNIKFADDLNQLTGYKKPASRELKVTFFP
Length: 49
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 14
E-value: 1960.349231111099

0    DLV  2  
     |||     
125  DLV  127


